### DISCUSSION TITLE
Importing into svelte.config.js

### DISCUSSION THREAD

**Discussion by k-dahl** [sveltejs/kit#12461] [Q&A]
Hey,

I am trying to programmatically create the `entries` for my svelte.config.js, as most of my pages are generated based on an array of data that lives in `$lib/data` - it's a very simple routine to generate the slugs for the `entries`, but the problem is that I cannot import my routine (or data) from `$lib/data` in svelte.config.js, as $lib does not appear to work there.

Is there another way I can import that data without using $lib? I have tried every variant of relative and absolute path I can think of to just point it to the dir manually, but all fail saying it cannot find the package.

Thanks for any insights!

**Comment by david-plugge** [comment]
I dont understand why you would have to do that actually. Can you provide some example code
